src/IncludedCommands/BasicCommands/createteam.lua

```diff
@@ -47,12 +52,12 @@ function Command:Run(CommandContext,Colors)
             local NewTeam = Instance.new("Team")
             NewTeam.TeamColor = Color
             NewTeam.AutoAssignable = false
-            NewTeam.Name = tostring(Color).." Team"
+            NewTeam.Name = Name
```

Comment:
This line is not safe to use because it accepts the name unfiltered. Paired with the request for the name to be optional, a better line would be `NewTeam.Name = (Name == nil and  tostring(Color).." Team" or self.API.Filter:FilterString(Name, CommandContext.Executor))"`. `FilterString` handles error handling internally and does not need to be wrapped in a `pcall`.